src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/BlockChunkFinder.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.plugins.workflow.graphanalysis;
+
+import org.jenkinsci.plugins.workflow.graph.BlockEndNode;
+import org.jenkinsci.plugins.workflow.graph.BlockStartNode;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Matches start and end of a block.  Any block
+ * @author Sam Van Oort
```

Comment:
Looks like per @jglick convention in this plugin is not to use @since so I'm off the hook... this time.